packages/tldraw/src/lib/shapes/text/TextArea.tsx

```diff
@@ -0,0 +1,49 @@
+import { stopEventPropagation } from '@tldraw/editor'
+
+export function TextArea({
+	rInput,
+	text,
+	handleFocus,
+	handleChange,
+	handleKeyDown,
+	handleBlur,
+	handleInputPointerDown,
+	handleDoubleClick,
+}: {
+	rInput: React.RefObject<HTMLTextAreaElement>
```

Comment:
forward ref instead here?

Reply:
(I mean they're taking it out in a few months anyway, but for now!)

Reply:
hah doh, not sure what i was smoking here - reworked!